.github/workflows/arma.yml

```diff
@@ -35,14 +35,13 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - name: Checkout the source code
-      uses: actions/checkout@master
-    - name: Build using HEMTT
-      uses: arma-actions/hemtt@master
-      with:
-        command: build --release --ci
+    - uses: actions/checkout@v3
+    - uses: arma-actions/hemtt@v1
+      name: Setup HEMTT
+    - name: Build
+      run: hemtt build
     - name: Upload Artifact
-      uses: actions/upload-artifact@v2-preview
+      uses: actions/upload-artifact@v3
       with:
-        name: ace3-${{ github.sha }}-nobin
-        path: releases/ace3_*.zip
+        name: ace3-${{ github.sha }}
```

Comment:
This will still be no bin, as binarize isn't available on Linux

Reply:
```suggestion
        name: ace3-${{ github.sha }}-nobin
```

Reply:
Should we run it on Windows instead then? Can use same prepare step as the pboproject job to have binarize etc setup

Reply:
right now we really just want it for the linting

but we could also use it to have a dev/master full build
we'd also need to enable script compiling